similarity.js

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env node
+
+//const distance = require('leven');
```

Comment:
Sure thing! I just did it up super fast, but your point is fair. Can get to it later this week!